src/posts/paper/99-changelog.ts

```diff
@@ -7,6 +7,11 @@ interface Change {
 }
 
 const changes: Change[] = [
+  {
+    date: new Date(2021, 0, 27),
+    content: `
+* Created new mask categories and updated multipliers for mask types. See Increased One-time interraction transmission rate from 6% to 9% to account for more contagious COVID variant B117. See [the Masks section of Research Sources](/paper/14-research-sources#masks) in the White Paper for details.`,
```

Comment:
I think you've combined the B117 update with the masks update. But they need two separate entries on different dates, yeah?

Reply:
Looks like copy paste with incomplete rewrite?

Reply:
resolved.